homeassistant/components/mealie/__init__.py

```diff
@@ -75,3 +92,8 @@ async def async_setup_entry(hass: HomeAssistant, entry: MealieConfigEntry) -> bo
 async def async_unload_entry(hass: HomeAssistant, entry: MealieConfigEntry) -> bool:
     """Unload a config entry."""
     return await hass.config_entries.async_unload_platforms(entry, PLATFORMS)
+
+
+def create_version(version: str) -> AwesomeVersion:
+    """Convert beta versions to PEP440."""
+    return AwesomeVersion(version.replace(BETA_STRATEGY_MEALIE, BETA_STRATEGY_PEP440))
```

Comment:
We don't have to create another Awesome version object for comparison.

You can do `1.0.0 < MIN_REQUIRED_MEALIE_V` directly

Reply:
No longer valid, as Awesomversion doesn't support the format indeed.